inbm/telemetry-agent/telemetry/telemetry_handling.py

```diff
@@ -303,7 +303,7 @@ def get_query_related_info(option: str, info: Dict) -> Dict:
                 del info['values'][elem]
     elif option == "version":
         info = {}
-        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()        
+        info['INBMVersionCommit'] = get_friendly_inbm_version_commit()
     elif option == "all":
```

Comment:
**<h2>$\color{#FFD700}Coding Style: CodeDocumentation$</h2>**
**Suggestion:** For the TelemetryTimer class, consider adding docstrings to explain the purpose and usage of each method.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
                class TelemetryTimer:
                    ...
                    def wait_collect(self, max_sleep_time: float = 0) -> None:
                        """
                        Sleep until time to collect. If max time is not specified, use the collection time.
                        @param max_sleep_time: Maximum time to sleep
                        """
                        ...
```

Reply:
good suggestion, but out of scope -- this is code reformatting